vagrant/provisioning/roles/license-manager/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: ensure group arkcase exists
+  become: yes
+  group:
+    name: arkcase
+    state: present
+
+- name: arkcase user
+  become: yes
+  user:
+    name: arkcase
+    shell: /bin/bash
+    group: arkcase
+    append: yes
+
+- name: deploy license-manager
+  include_tasks: "{{ role_path }}/../arkcase-microservice/tasks/deploy_microservice.yml"
+  vars:
+    microservice:
+      name: license-manager
+      version: "{{ license_manager_version | default(arkcase_version) }}"
+      java_opts: "-Dserver.port=8088 -Dspring.cloud.config.uri=https://{{ config_server_host | default(internal_host) }}/config -Dspring.profiles.include=server,runtime -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.keyStore={{ java_key_store }} -Djavax.net.ssl.trustStore={{ java_trust_store}} -Djavax.net.ssl.keyStorePassword={{ java_key_store_pass }} -Dlogging.file.name={{ root_folder }}/log/license-manager/license-manager.log -Djava.io.tmpdir={{ root_folder }}/data/license-manager/tmp"
```

Comment:
typically we include the ArkCase extension name in the `spring.profiles.include`, e.g. `-Dspring.profiles.include={{ arkcase_portal_extension ~ ',' if arkcase_portal_extension != '' else '' }}server,runtime`